community-operators/node-problem-detector/node-problem-detector.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,171 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
```

Comment:
Is there an example of a NodeProblemDetector CR? If so this should be added to `metadata.annotations.alm-examples` as stringified JSON.

Reply:
Added. Thanks